### PR TITLE
doc fixes for ansible deployments

### PR DESCRIPTION
**Pull request by Deflaimun**
Restores PR https://github.com/redpanda-data/documentation-private/pull/2083

### CODE REVIEW

**Review by gene-redpanda — APPROVED** [review]
Thanks a ton!

File: modules/deploy/pages/deployment-option/self-hosted/manual/production/production-deployment-automation.adoc [hunk 1]
```diff
@@ -146,7 +114,13 @@ GCP::
 cd gcp
 ----
 
-. You need an existing subnet in which to deploy the virtual machines (VMs). The subnet's attached firewall should allow inbound traffic on ports 22, 3000, 8082, 8888, 8889, 9090,  9092, 9644, and 33145. This module adds the `rp-node` tag to the deployed VMs, which can be used as the target tag for the firewall rule.
+. Set GCP credentials. Terraform provides multiple ways to set the AWS secret and key. See the 
+https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/getting_started[Terraform documentation^].
+
+. Our module will generate a properly configured GCP network within your GCP project by default. You can disable this functionality by commenting out that section of the main.tf and providing your own subnet to var.subnet.
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/self-hosted/manual/production/production-deployment-automation.adoc]:
```suggestion
. This module generates a properly-configured GCP network within your GCP project by default. You can disable this functionality by commenting out that section of the main.tf and providing your own subnet to var.subnet. 
```

File: modules/deploy/pages/deployment-option/self-hosted/manual/production/production-deployment-automation.adoc [hunk 2]
```diff
@@ -160,59 +134,25 @@ terraform init
 terraform apply
 ----
 
-The following example creates a three-broker cluster using the subnet named `redpanda-cluster-subnet`:
+The following example creates a three-broker cluster
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/self-hosted/manual/production/production-deployment-automation.adoc]:
```suggestion
The following example shows how to creates a three-broker cluster:
```

File: modules/deploy/pages/deployment-option/self-hosted/manual/production/production-deployment-automation.adoc [hunk 3]
```diff
@@ -160,59 +134,25 @@ terraform init
...
-| `public_key_path`
-| Path to the public key of the keypair used to access the broker.
-
-| `ssh_user`
-| The ssh user. Must match the one in the public ssh key's comments.
-|===
-
---
-=====
+For a full and up to date list of configuration options please see the (https://registry.terraform.io/modules/redpanda-data/redpanda-cluster/gcp/latest[Terraform module^]):
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/self-hosted/manual/production/production-deployment-automation.adoc]:
```suggestion
For a full, up-to-date list of configuration options, see the (https://registry.terraform.io/modules/redpanda-data/redpanda-cluster/gcp/latest[Terraform module^]):
```

**Review by Feediver1 — APPROVED** [review]

**Pull request merged**